packages/office-ui-fabric-react/src/components/Rating/Rating.test.tsx

```diff
@@ -78,6 +78,78 @@ describe('Rating', () => {
     expect((ratingInputs[4] as HTMLInputElement).checked).to.be.eq(true);
   });
 
+  it('Deciamal Rating not aggregate', () => {
```

Comment:
The tests should read like this:

```ts
describe('Thing', () => {
  it('does things', () => {});
})
```
So that it read like "Thing does things". In other words, your "it" descriptions should follow the convention seen in other tests.

Also you have spelling errors here.